update entry point articles in ag for android (#646)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/features/protection/protection.md

:::

The *Protection* tab includes the following features or settings. You’ll find a detailed article for each of them in this Knowledge Base.

## Ad blocking

Settings to block ads in browsers and apps

[Ad blocking](/adguard-for-android/features/protection/ad-blocking.md)

## Tracking protection

Settings that allow you to block tracking tools that collect user information

[Tracking protection](/adguard-for-android/features/protection/tracking-protection.md)

## Annoyance blocking

> Settings to block annoyances such as cookie consent pop-ups, social widgets, and banners that prompt you to download mobile apps

Reviewer: ```suggestion
Settings to block annoyances such as cookie consent popups, social widgets, and banners that prompt you to download mobile apps
```

[Annoyance blocking](/adguard-for-android/features/protection/annoyance-blocking.md)

## DNS protection

> Settings for a secure DNS server and DNS filters to block ads, trackers and dangerous websites

Reviewer: ```suggestion
Settings for a secure DNS server and DNS filters to block ads, trackers, and dangerous websites
```

[DNS protection](/adguard-for-android/features/protection/dns-protection.md)

## Firewall

> Feature that allows you to control Internet access for individual apps

Reviewer: ```suggestion
A feature that allows you to control Internet access for individual apps
```

[Firewall](/adguard-for-android/features/protection/firewall/firewall.md)

## Browsing security

> Feature that blocks access to malicious and phishing websites and protects you from online threats

Reviewer: ```suggestion
A feature that blocks access to malicious and phishing websites and protects you from online threats
```
